pygmt/helpers/caching.py

```diff
@@ -100,4 +89,5 @@ def cache_data():
         "@tut_ship.xyz",
         "@usgs_quakes_22.txt",
     ]
-    which(fname=datasets, download="a")
+
+    which(fname=datasets + caches + tiles, download="a")
```

Comment:
Can we continue to use just a single list?

Reply:
We can. I just feel that having three separate lists makes it easier to organize the file list.

Reply:
Well, codecov is warning that there's missing code coverage for the extra two lists :slightly_smiling_face:

Reply:
That's a good point to push me to merge them into a single list.

Reply:
Done in 72fcaa1b18ddb922005c0dc482788d0e439098cc.